Unix/pal/cpu.c

```diff
@@ -10,6 +10,23 @@
 #include <sys/time.h>
 #include "cpu.h"
 
+#if defined(CONFIG_OS_FREEBSD)
+long timezone_calculated = 0;
+
+__attribute__((constructor)) static void InitializeTimeOffset(void)
+{
+    // Unlike SYSV, BSD does not have timezone with time offset. (It returns char* with name)
+    // We need to calculate it once.
+    struct tm gmt;
+    struct tm local;
+
+    time_t now = time(NULL);
```

Comment:
where is now use?

Reply:
look at CPU_GetLocalTimestampz() @JumpingYang001. On Linux, timezone is  extern long with offset to GMT. BSD does not have that. (or OSX/MacOS) Since I failed to figure out what we do for Mac, I simply calculate once difference between GMT and local.

Reply:
yeah, I found there is a [char* timezone() function](https://github.com/freebsd/freebsd/blob/master/include/time.h#L197) in FreeBSD.

I think OSX should use exist logic, I don't know much about how BSD dealt with it.
I just surprise the `now` variable here isn't used?
:)

Reply:
I am not sure if my condition is good, just a  thought. :)